### ISSUE TITLE
new map: schenken / tauschen

### ISSUE THREAD

**Issue by species** [TransforMap/transfor-map#38]
was wished on Mapping Meeting Munich, June 18-19, see protocol: http://dossier.allmende.io/15MMMunich

**Comment by pmackay** [comment]
So this is gift/trade economy? What kinds of content would this show, different to other maps?

**Comment by species** [comment]
Yes, this should be about gifting/bartering. Or the “Freegan”'s Map ;-)

It would show places where you can get things for free, e.g. public drinking water fountains, public bookshelves, talent&time barter clubs, give-away shops, repair-cafes, pay-what-you-want restaurants or public fruit trees.

I am open for suggestions of what else should be shown here!
